src/sync/partial_sync.hpp

```diff
@@ -19,20 +19,35 @@
 #ifndef REALM_OS_PARTIAL_SYNC_HPP
 #define REALM_OS_PARTIAL_SYNC_HPP
 
+#include "impl/results_notifier.hpp"
+
 #include <functional>
 #include <memory>
 #include <string>
 
 namespace realm {
-
 class Realm;
 class Results;
+class Group;
 
 namespace partial_sync {
+	enum class SubscriptionState : int8_t;
```

Comment:
Namespace body shouldn't be indented like this.

Indentation, when needed, should use spaces rather than tabs.